src/main/java/com/booster/server/NoteController.java

```diff
@@ -0,0 +1,28 @@
+package com.booster.server;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@Slf4j
+@RestController
+@RequestMapping("/note")
+@RequiredArgsConstructor
+public class NoteController {
```

Comment:
No need for a `public` modifier since the controller is not directly used anywhere